ngraph/frontend/tensorflow/src/op/crop_and_resize.cpp

```diff
@@ -0,0 +1,142 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "op_table.hpp"
+#include "openvino/opsets/opset8.hpp"
+
+using namespace std;
+using namespace ov::opset8;
+
+namespace ov {
+namespace frontend {
+namespace tf {
+namespace op {
+
+OutputVector translate_crop_and_resize_op(const NodeContext& node) {
+    /// ng_input: [batch, image_height, image_width, depth]
```

Comment:
please add `TODO: re-write this code to use partial shapes`

Reply:
we already have exceptions here.
`.get_partial_shape().is_static(),
                           "Dynamic shapes are not supported.");`
not sure it's possible to re-write this code to support dynamism, let's figure it out in a more detailed review